ktor-client/ktor-client-features/ktor-client-websockets/common/test/io/ktor/client/tests/features/WebSocketRemoteTest.kt

```diff
@@ -13,7 +13,7 @@ import kotlinx.coroutines.*
 import kotlin.test.*
 
 class WebSocketRemoteTest : ClientLoader() {
-    private val echoWebsocket = "echo.websocket.org"
+    private val echoWebsocket = "$TEST_WEBSOCKET_SERVER/websockets/echo"
```

Comment:
is it related?

Reply:
Can extract in the separate PR if needed